cg/services/bcl_convert_metrics_service/bcl_convert_metrics_service.py

```diff
@@ -101,21 +104,29 @@ def create_sample_run_metrics(
         )
         sample_id: int = store.get_sample_by_internal_id(sample_internal_id).id
 
+        yield_: float = metrics_parser.get_yield_for_sample_in_lane(
+            sample_internal_id=sample_internal_id, lane=lane
+        )
+        yield_q30: float = metrics_parser.get_yield_q30_for_sample_in_lane(
+            sample_internal_id=sample_internal_id, lane=lane
+        )
+
         return IlluminaSampleSequencingMetrics(
-            run_metrics_id=run_metrics_id,
             sample_id=sample_id,
             type=DeviceType.ILLUMINA,
             flow_cell_lane=lane,
             total_reads_in_lane=total_reads,
             base_percentage_passing_q30=q30_bases_percent,
             base_mean_quality_score=mean_quality_score,
+            _yield=yield_,
```

Comment:
Is this change in the model already in place?
```suggestion
            yield_=yield_,
```

Reply:
Yes, this field is updated in the model. It does not require an alembic migration though since in the database it is listed as "yield" as per __name_pos parameter that is provided there.